x-pack/legacy/plugins/siem/package.json

```diff
@@ -1,16 +1,10 @@
 {
   "author": "Elastic",
-  "name": "siem",
+  "name": "siem-legacy-ui",
```

Comment:
This change is just to explicitly state that the client UI is still on the old platform (and disambiguate from the non-legacy package.json), correct? Are there any downstream effects of changing the app name here, or does all of the magic that keys off app name/id/title happen from what is set in legacy `register_feature.ts`?

https://github.com/elastic/kibana/blob/27f5eaaaab35dca1c6c90acdf1cc96b10e2f6d5c/x-pack/legacy/plugins/siem/public/register_feature.ts#L11-L14

Reply:
@spong correct, the name here is just to disambiguate; there's a check in the build process that actually verifies that each package.json has a unique name. I have not seen anything to indicate that this name affects the build process but I'll verify that with platform.